atlasdb-config/src/main/java/com/palantir/atlasdb/config/AtlasDbConfig.java

```diff
@@ -254,6 +254,15 @@ public int getDefaultLockTimeoutSeconds() {
         return AtlasDbConstants.DEFAULT_LOCK_TIMEOUT_SECONDS;
     }
 
+    /**
+     * The maximum number of threads to use in concurrent getRanges requests when the user
+     * does not explicitly set a value.
+     */
+    @Value.Default
+    public int defaultGetRangesConcurrency() {
```

Comment:
It is confusing to have this on the standard config and not the KVS config. Is there any reason for this one to be here instead?

Reply:
Yeah wasn't totally sure which made more sense. Initially was thinking this could be global but the more I think about I'm thinking we could end up changing it based on KVS, so will move.